src/core/weights/edgeWeights/edgeWeights.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import {type EdgeAddressT} from "../../graph";
+import {edgeWeightEvaluator} from "../../algorithm/weightEvaluator";
+import {orElse as either} from "../../../util/null";
+import {
+  type EdgeWeightsT,
+  type EdgeOperator,
+  type EdgeWeight,
+  empty,
+  copy,
+  merge,
+} from "./edgeWeightsT";
+
```

Comment:
```suggestion
/**
 * A higher level interface on EdgeWeightsT that includes helper functions
 */
```

Reply:
Let's add a similar comment for NodeWeights too

Reply:
✅